### ISSUE TITLE
Plugin could not be activated because it triggered a fatal error.

### ISSUE THREAD

**Issue by joesawa** [robertdevore/benchpress#1]
Screenshot of report attached. Using WPEngine, php 7.4

```
Parse error: syntax error, unexpected '=>' (T_DOUBLE_ARROW) in /nas/content/live/wwwuatbestegg/wp-content/plugins/benchpress-main/includes/helper-functions.php on line 43
```

![Screenshot 2024-11-11 at 10 35 05 AM](https://github.com/user-attachments/assets/f1825a19-b17d-4aad-ab84-8c4791e61ff5)

**Comment by robertdevore** [comment]
Thank you for letting me know. There's a benchmark built in that requires PHP 8.0 so I'm going to push an update that disables that from the settings and removes it from the benchmarks on PHP 7.4.0

I'll get that out later this evening 🤘

**Comment by robertdevore** [comment]
@joesawa  This has been fixed in v1.0.1 💪
